src/pretix/static/pretixcontrol/js/ui/editor.js

```diff
@@ -553,8 +553,16 @@ var editor = {
             $("#toolbox-content-other-help").toggle($("#toolbox-content").val() === "other" || $("#toolbox-content").val() === "other_i18n");
             o.content = $("#toolbox-content").val();
             if ($("#toolbox-content").val() === "other") {
+                if (this.id === "toolbox-content") {
```

Comment:
- This is the first code introducing ``this`` in this function, which I'm not sure of whether it is a good idea. First of all, as a reader, I had to think quite a while what ``this`` is here since we're not *directly* in an event handler. This could be solved with a comment.
- This will also *overwrite* values of the other field if there currently was one. Should we only call val() if it was previously empty? Not sure what would be expected behaviour from users here, to be honest, we can also keep it like this.

Reply:
I thought about using `event.target` instead of `this`, but as you said, we are not _directly_ in an event handler. So yes, a comment is a must here!

Regarding overwrite: yes, it will overwrite, but the content in there would be the text of the last content-other object selected – so not really helpful. Also, it should actually be empty anyways (see changes in Line 639 ff).

Reply:
👍

Reply:
I actually changed it to `e.target` – `_update_values_from_toolbox` is bound to several events and only once used internally, which I passed the event to it. I also updated the comment.